compiler/rustc_macros/src/session_diagnostic.rs

```diff
@@ -105,8 +104,17 @@ impl SessionDiagnosticDeriveError {
     }
 }
 
-fn span_err(span: impl proc_macro::MultiSpan, msg: &str) -> proc_macro::Diagnostic {
-    Diagnostic::spanned(span, proc_macro::Level::Error, msg)
+macro_rules! with_help {
```

Comment:
Does this need to be a macro? This seems like it can be a normal function, like it was before.

Reply:
I can add a comment, but the macro is intended to be temporary until a bootstrap takes place, hence the implementation. The method name changed but the signature stayed the same, so it was easier to just do this.